### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by echarlaix**

### CODE REVIEW

File: docs/source/quickstart.mdx [hunk 1]
```diff
@@ -62,6 +68,7 @@ Similarly, you can apply static quantization by simply setting `is_static` to `T
 Static quantization relies on feeding batches of data through the model to estimate the activation quantization parameters ahead of inference time. To support this, 🤗 Optimum allows you to provide a _calibration dataset_. The calibration dataset can be a simple `Dataset` object from the 🤗 Datasets library, or any dataset that's hosted on the Hugging Face Hub. For this example, we'll pick the [`sst2`](https://huggingface.co/datasets/glue/viewer/sst2/test) dataset that the model was originally trained on:
 
 ```python
+>>> from functools import partial
```
Comment (regisss) [inline docs/source/quickstart.mdx:71]: It does not seem to be used anywhere in the following lines

**Review by regisss — APPROVED** [review]
Thanks for this PR @echarlaix !!
Just a few nits.

File: docs/source/quickstart.mdx (hunk 1 above)
Comment (echarlaix) [reply docs/source/quickstart.mdx:71]: Used when calling the `get_calibration_dataset` method of the quantizer

File: README.md [hunk 2]
```diff
@@ -65,49 +65,65 @@ python -m pip install git+https://github.com/huggingface/optimum.git#egg=optimum
...
+
+# Save the onnx model and tokenizer
+model.save_pretrained(save_directory, file_name=file_name)
+tokenizer.save_pretrained(save_directory)
+```
 
 ### Quantization
 
-For example, here's how you can apply dynamic quantization with ONNX Runtime:
+Now let's see now how we can apply dynamic quantization with ONNX Runtime:
```
[the remark this answers is not in the document]
Comment (echarlaix) [reply README.md:95]: Thanks for the suggestion, I will modify the current version !

**Comment by echarlaix** [comment]
Thanks for the quick review @regisss

**Pull request merged**